apps/vlasov.c

```diff
@@ -1092,7 +1102,7 @@ comm_reduce_app_stat(const gkyl_vlasov_app* app,
   };
 
   int64_t l_red_global[L_END];
-  gkyl_comm_allreduce(app->comm, GKYL_INT_64, GKYL_MAX, L_END, l_red, l_red_global);
+  gkyl_comm_allreduce_host(app->comm, GKYL_INT_64, GKYL_MAX, L_END, l_red, l_red_global);
```

Comment:
Making a note here in this PR that this fixes Vlasov on GPUs in gk-g0-app, though we are currently refactoring the comm object as part of the multi-block work here: https://github.com/ammarhakim/gkylzero/tree/multib-app so that there will not be "host" methods.